apps/expo-go/ios/Exponent/Kernel/AppLoader/CachedResource/EXManifestResource.m

```diff
@@ -202,13 +204,31 @@ - (BOOL)_isManifestVerificationBypassed: (id) manifestObj
 
 - (NSInteger)sdkVersionStringToInt:(nonnull NSString *)sdkVersion {
   NSRange snackSdkVersionRange = [sdkVersion rangeOfString: @"."];
-  return [[sdkVersion substringToIndex: snackSdkVersionRange.location] intValue];
+  return [[sdkVersion substringToIndex:snackSdkVersionRange.location] intValue];
 }
 
-- (NSString *)supportedSdkVersionsConjunctionString:(nonnull NSString *)conjuction {
+-(NSMutableArray *)supportedSdkVersionInts {
```

Comment:
Returning mutable arrays/dicts is not a good practice if there is no strong need to do so.
```suggestion
- (NSArray<NSNumber *> *)supportedSdkVersionInts
{
```